+ source /root/mugen-riscv/libs/locallibs/common_lib.sh
++ python3 --version
++ '[' 0 -eq 0 ']'
++ source /root/mugen-riscv/libs/locallibs/common_lib_python.sh
+ main
++ type -t post_test
+ '[' -n function ']'
+ trap post_test EXIT INT HUP TERM
+ rpm -qa
+ grep expect
++ type -t config_params
+ '[' -n '' ']'
++ type -t pre_test
+ '[' -n function ']'
+ pre_test
+ LOG_INFO 'Start to prepare the test environment.'
+ message='Start to prepare the test environment.'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level info --message 'Start to prepare the test environment.'
Fri Nov 11 21:48:40 2022 - INFO  - Start to prepare the test environment.
+ DNF_INSTALL 'freeradius freeradius-utils'
+ pkgs='freeradius freeradius-utils'
+ node=1
+ '[' -z '' ']'
+ tmpfile=
++ python3 /root/mugen-riscv/libs/locallibs/rpm_manage.py install --pkgs 'freeradius freeradius-utils' --node 1 --tempfile ''
+ tmpfile2='Last metadata expiration check: 0:18:21 ago on Fri 11 Nov 2022 09:30:56 PM CST.
Error: 
 Problem: conflicting requests
  - nothing provides perl(Net::IP) needed by freeradius-utils-3.0.25-1.oe2203.riscv64
(try to add '\''--skip-broken'\'' to skip uninstallable packages or '\''--nobest'\'' to use not only best candidate packages)'
+ '[' -z '' ']'
+ tmpfile='Last metadata expiration check: 0:18:21 ago on Fri 11 Nov 2022 09:30:56 PM CST.
Error: 
 Problem: conflicting requests
  - nothing provides perl(Net::IP) needed by freeradius-utils-3.0.25-1.oe2203.riscv64
(try to add '\''--skip-broken'\'' to skip uninstallable packages or '\''--nobest'\'' to use not only best candidate packages)'
+ echo 'steve  Cleartext-Password := "testing"
       Service-Type = Framed-User,
       Framed-Protocol = PPP,
       Framed-IP-Address = 172.16.3.33,
       Framed-IP-Netmask = 255.255.255.0,
       Framed-Routing = Broadcast-Listen,
       Framed-Filter-Id = "std.ppp",
       Framed-MTU = 1500,
       Framed-Compression = Van-Jacobsen-TCP-IP
    '
oe_test_freeradius_freeradius-utils_radtestAndRadwho.sh: line 34: /etc/raddb/users: No such file or directory
+ systemctl start radiusd
Failed to start radiusd.service: Unit radiusd.service not found.
+ SLEEP_WAIT 2
+ wait_time=2
+ cmd=
+ mode=1
+ python3 /root/mugen-riscv/libs/locallibs/sleep_wait.py --time 2 --cmd '' --mode 1
+ LOG_INFO 'End to prepare the test environment.'
+ message='End to prepare the test environment.'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level info --message 'End to prepare the test environment.'
Fri Nov 11 21:49:24 2022 - INFO  - End to prepare the test environment.
++ type -t run_test
+ '[' -n function ']'
+ run_test
+ LOG_INFO 'Start to run test.'
+ message='Start to run test.'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level info --message 'Start to run test.'
Fri Nov 11 21:49:26 2022 - INFO  - Start to run test.
+ radtest -d /etc/raddb/ steve testing localhost 0 testing123
+ grep Access-Accept
oe_test_freeradius_freeradius-utils_radtestAndRadwho.sh: line 44: radtest: command not found
+ CHECK_RESULT 1 0 0 'radtest -d execution failed.'
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log='radtest -d execution failed.'
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n 'radtest -d execution failed.'
+ LOG_ERROR 'radtest -d execution failed.'
+ message='radtest -d execution failed.'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'radtest -d execution failed.'
Fri Nov 11 21:49:28 2022 - ERROR - radtest -d execution failed.
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_freeradius_freeradius-utils_radtestAndRadwho.sh line 45'
+ message='oe_test_freeradius_freeradius-utils_radtestAndRadwho.sh line 45'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'oe_test_freeradius_freeradius-utils_radtestAndRadwho.sh line 45'
Fri Nov 11 21:49:30 2022 - ERROR - oe_test_freeradius_freeradius-utils_radtestAndRadwho.sh line 45
+ return 0
+ radtest -t pap steve testing localhost 0 testing123
+ grep Access-Accept
oe_test_freeradius_freeradius-utils_radtestAndRadwho.sh: line 46: radtest: command not found
+ CHECK_RESULT 1 0 0 'radtest -t execution failed.'
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log='radtest -t execution failed.'
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n 'radtest -t execution failed.'
+ LOG_ERROR 'radtest -t execution failed.'
+ message='radtest -t execution failed.'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'radtest -t execution failed.'
Fri Nov 11 21:49:32 2022 - ERROR - radtest -t execution failed.
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_freeradius_freeradius-utils_radtestAndRadwho.sh line 47'
+ message='oe_test_freeradius_freeradius-utils_radtestAndRadwho.sh line 47'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'oe_test_freeradius_freeradius-utils_radtestAndRadwho.sh line 47'
Fri Nov 11 21:49:33 2022 - ERROR - oe_test_freeradius_freeradius-utils_radtestAndRadwho.sh line 47
+ return 0
+ radtest -P udp steve testing localhost 0 testing123
+ grep Access-Accept
oe_test_freeradius_freeradius-utils_radtestAndRadwho.sh: line 48: radtest: command not found
+ CHECK_RESULT 1 0 0 'radtest -P execution failed.'
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log='radtest -P execution failed.'
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n 'radtest -P execution failed.'
+ LOG_ERROR 'radtest -P execution failed.'
+ message='radtest -P execution failed.'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'radtest -P execution failed.'
Fri Nov 11 21:49:35 2022 - ERROR - radtest -P execution failed.
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_freeradius_freeradius-utils_radtestAndRadwho.sh line 49'
+ message='oe_test_freeradius_freeradius-utils_radtestAndRadwho.sh line 49'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'oe_test_freeradius_freeradius-utils_radtestAndRadwho.sh line 49'
Fri Nov 11 21:49:37 2022 - ERROR - oe_test_freeradius_freeradius-utils_radtestAndRadwho.sh line 49
+ return 0
+ radtest -x steve testing localhost 0 testing123
oe_test_freeradius_freeradius-utils_radtestAndRadwho.sh: line 50: radtest: command not found
+ grep Access-Accept
+ CHECK_RESULT 1 0 0 'radtest -x execution failed.'
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log='radtest -x execution failed.'
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n 'radtest -x execution failed.'
+ LOG_ERROR 'radtest -x execution failed.'
+ message='radtest -x execution failed.'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'radtest -x execution failed.'
Fri Nov 11 21:49:39 2022 - ERROR - radtest -x execution failed.
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_freeradius_freeradius-utils_radtestAndRadwho.sh line 51'
+ message='oe_test_freeradius_freeradius-utils_radtestAndRadwho.sh line 51'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'oe_test_freeradius_freeradius-utils_radtestAndRadwho.sh line 51'
Fri Nov 11 21:49:41 2022 - ERROR - oe_test_freeradius_freeradius-utils_radtestAndRadwho.sh line 51
+ return 0
+ radtest -4 steve testing localhost 0 testing123
+ grep Access-Accept
oe_test_freeradius_freeradius-utils_radtestAndRadwho.sh: line 52: radtest: command not found
+ CHECK_RESULT 1 0 0 'radtest -4 execution failed.'
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log='radtest -4 execution failed.'
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n 'radtest -4 execution failed.'
+ LOG_ERROR 'radtest -4 execution failed.'
+ message='radtest -4 execution failed.'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'radtest -4 execution failed.'
Fri Nov 11 21:49:42 2022 - ERROR - radtest -4 execution failed.
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_freeradius_freeradius-utils_radtestAndRadwho.sh line 53'
+ message='oe_test_freeradius_freeradius-utils_radtestAndRadwho.sh line 53'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'oe_test_freeradius_freeradius-utils_radtestAndRadwho.sh line 53'
Fri Nov 11 21:49:44 2022 - ERROR - oe_test_freeradius_freeradius-utils_radtestAndRadwho.sh line 53
+ return 0
+ radtest -6 steve testing localhost 0 testing123
+ grep '\['
oe_test_freeradius_freeradius-utils_radtestAndRadwho.sh: line 54: radtest: command not found
+ CHECK_RESULT 1 0 0 'radtest -6 execution failed.'
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log='radtest -6 execution failed.'
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n 'radtest -6 execution failed.'
+ LOG_ERROR 'radtest -6 execution failed.'
+ message='radtest -6 execution failed.'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'radtest -6 execution failed.'
Fri Nov 11 21:49:46 2022 - ERROR - radtest -6 execution failed.
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_freeradius_freeradius-utils_radtestAndRadwho.sh line 55'
+ message='oe_test_freeradius_freeradius-utils_radtestAndRadwho.sh line 55'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'oe_test_freeradius_freeradius-utils_radtestAndRadwho.sh line 55'
Fri Nov 11 21:49:48 2022 - ERROR - oe_test_freeradius_freeradius-utils_radtestAndRadwho.sh line 55
+ return 0
+ systemctl stop radiusd
Failed to stop radiusd.service: Unit radiusd.service not loaded.
+ touch /var/log/radius/radutmp
touch: cannot touch '/var/log/radius/radutmp': No such file or directory
+ radwho -c
oe_test_freeradius_freeradius-utils_radtestAndRadwho.sh: line 58: radwho: command not found
+ grep Name
+ CHECK_RESULT 1 0 0 'radwho -c execution failed.'
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log='radwho -c execution failed.'
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n 'radwho -c execution failed.'
+ LOG_ERROR 'radwho -c execution failed.'
+ message='radwho -c execution failed.'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'radwho -c execution failed.'
Fri Nov 11 21:49:50 2022 - ERROR - radwho -c execution failed.
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_freeradius_freeradius-utils_radtestAndRadwho.sh line 59'
+ message='oe_test_freeradius_freeradius-utils_radtestAndRadwho.sh line 59'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'oe_test_freeradius_freeradius-utils_radtestAndRadwho.sh line 59'
Fri Nov 11 21:49:51 2022 - ERROR - oe_test_freeradius_freeradius-utils_radtestAndRadwho.sh line 59
+ return 0
+ radwho -d /etc/raddb/
+ grep Name
oe_test_freeradius_freeradius-utils_radtestAndRadwho.sh: line 60: radwho: command not found
+ CHECK_RESULT 1 0 0 'radwho -d execution failed.'
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log='radwho -d execution failed.'
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n 'radwho -d execution failed.'
+ LOG_ERROR 'radwho -d execution failed.'
+ message='radwho -d execution failed.'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'radwho -d execution failed.'
Fri Nov 11 21:49:53 2022 - ERROR - radwho -d execution failed.
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_freeradius_freeradius-utils_radtestAndRadwho.sh line 61'
+ message='oe_test_freeradius_freeradius-utils_radtestAndRadwho.sh line 61'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'oe_test_freeradius_freeradius-utils_radtestAndRadwho.sh line 61'
Fri Nov 11 21:49:54 2022 - ERROR - oe_test_freeradius_freeradius-utils_radtestAndRadwho.sh line 61
+ return 0
+ radwho -F /var/log/radius/radutmp
+ grep Name
oe_test_freeradius_freeradius-utils_radtestAndRadwho.sh: line 62: radwho: command not found
+ CHECK_RESULT 1 0 0 'radwho -F execution failed.'
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log='radwho -F execution failed.'
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n 'radwho -F execution failed.'
+ LOG_ERROR 'radwho -F execution failed.'
+ message='radwho -F execution failed.'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'radwho -F execution failed.'
Fri Nov 11 21:49:55 2022 - ERROR - radwho -F execution failed.
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_freeradius_freeradius-utils_radtestAndRadwho.sh line 63'
+ message='oe_test_freeradius_freeradius-utils_radtestAndRadwho.sh line 63'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'oe_test_freeradius_freeradius-utils_radtestAndRadwho.sh line 63'
Fri Nov 11 21:49:56 2022 - ERROR - oe_test_freeradius_freeradius-utils_radtestAndRadwho.sh line 63
+ return 0
+ LOG_INFO 'End to run test.'
+ message='End to run test.'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level info --message 'End to run test.'
Fri Nov 11 21:49:57 2022 - INFO  - End to run test.
+ CASE_RESULT 0
+ case_re=0
+ test -z 9
+ test 9 -gt 0
+ LOG_ERROR 'failed to execute the case.'
+ message='failed to execute the case.'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'failed to execute the case.'
Fri Nov 11 21:49:58 2022 - ERROR - failed to execute the case.
+ exit 9
+ post_test
+ LOG_INFO 'Start to restore the test environment.'
+ message='Start to restore the test environment.'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level info --message 'Start to restore the test environment.'
Fri Nov 11 21:49:59 2022 - INFO  - Start to restore the test environment.
+ DNF_REMOVE
+ node=1
+ pkg_list=
+ mode=0
+ [[ -z Last metadata expiration check: 0:18:21 ago on Fri 11 Nov 2022 09:30:56 PM CST.
Error: 
 Problem: conflicting requests
  - nothing provides perl(Net::IP) needed by freeradius-utils-3.0.25-1.oe2203.riscv64
(try to add '--skip-broken' to skip uninstallable packages or '--nobest' to use not only best candidate packages) ]]
+ '[' 0 -ne 0 ']'
+ '[' 1 == 0 ']'
+ python3 /root/mugen-riscv/libs/locallibs/rpm_manage.py remove --node 1 --pkgs '' --tempfile 'Last metadata expiration check: 0:18:21 ago on Fri 11 Nov 2022 09:30:56 PM CST.
Error: 
 Problem: conflicting requests
  - nothing provides perl(Net::IP) needed by freeradius-utils-3.0.25-1.oe2203.riscv64
(try to add '\''--skip-broken'\'' to skip uninstallable packages or '\''--nobest'\'' to use not only best candidate packages)'
Traceback (most recent call last):
  File "/root/mugen-riscv/libs/locallibs/rpm_manage.py", line 173, in <module>
    exitcode = rpm_remove(args.node, args.pkgs, args.tempfile)
  File "/root/mugen-riscv/libs/locallibs/rpm_manage.py", line 144, in rpm_remove
    with open(tmpfile, "r") as f:
OSError: [Errno 36] File name too long: "Last metadata expiration check: 0:18:21 ago on Fri 11 Nov 2022 09:30:56 PM CST.\nError: \n Problem: conflicting requests\n  - nothing provides perl(Net::IP) needed by freeradius-utils-3.0.25-1.oe2203.riscv64\n(try to add '--skip-broken' to skip uninstallable packages or '--nobest' to use not only best candidate packages)"
+ '[' 0 -ne 0 ']'
+ rm -rf /etc/raddb
+ rm -rf /var/log/radius
+ LOG_INFO 'End to restore the test environment.'
+ message='End to restore the test environment.'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level info --message 'End to restore the test environment.'
Fri Nov 11 21:50:04 2022 - INFO  - End to restore the test environment.
